content/influxdb/clustered/query-data/troubleshoot-and-optimize/report-query-performance-issues.md

```diff
@@ -141,14 +147,21 @@ As an example, consider the following test plan outline:
 4. Run 10 concurrent instances of Query A and allow the cluster to recover for 1 minute.
 5. Run 20 concurrent instances of Query A and allow the cluster to recover for 1 minute.
 6. Run 40 concurrent instances of Query A and allow the cluster to recover for 1 minute.
-7. Provide InfluxData the debug information [described below](#gather-debug-info).
+7. Provide InfluxData the debug information [described below](#gather-debug-information).
 
 {{% note %}}
 This is just an example. You don't have to go beyond the scale where queries get slower
 but you may also need to go further than what's outlined here.
 {{% /note %}}
 
-### Gather debug info
+### Capture dashboard screens
+
+If you have set up alerts and dashboards for monitoring your cluster, capture
+screenshots of dashboard events for Queriers, Compactors, and Ingesters.
+
+See [system query examples](/influxdb/clustered/admin/query-system-data/#system-query-examples).
```

Comment:
```suggestion
```
System query examples are different from dashboards, so I suggest removing the link here. 

Probably link to the page "Visualize data" instead: https://docs.influxdata.com/influxdb/clustered/process-data/visualize/?

Or you may want to ask the Clustered team about the dashboard

Reply:
This page is discussing information customer should collect when reporting query performance issues, so I don't think https://docs.influxdata.com/influxdb/clustered/process-data/visualize/ is a good choice since that section is largely telling the customer how to visualize data stored in InfluxDB.

But also, I think this link is out of place in the current section. It would be better to put this link somewhere under the `influxdb/clustered/query-data/troubleshoot-and-optimize/report-query-performance-issues/#gather-system-information` section since that's the section about system table data collection.

I think we don't currently have customer-facing documentation for the grafana dashboards this section would be referring to so to my knowledge there is nothing to link to here. The dashboards described here are copied over from Serverless/CST and are only available in Clustered if the customer enables the `grafana` feature flag. Until our next release, these dashboards are mostly broken so it's fine not to mention them in public docs yet. We'll just need to make sure there are appropriate github issues to follow-up with additional documentation after that release.

Reply:
Thanks for the feedback everyone. I'll hold off on mentioning dashboards for now--I didn't know if users were expected to build their own or if we provided configurations for them or ....